client/my-sites/stats/site.jsx

```diff
@@ -567,149 +568,157 @@ class StatsSite extends Component {
 						) }
 					</>
 
-					{ ! isOdysseyStats && <MiniCarousel slug={ slug } isSitePrivate={ isSitePrivate } /> }
-
-					<div className={ moduleListClasses }>
-						<StatsModuleTopPosts
-							moduleStrings={ moduleStrings.posts }
-							period={ this.props.period }
-							query={ query }
-							summaryUrl={ this.getStatHref( 'posts', query ) }
-							className={ halfWidthModuleClasses }
-						/>
-						<StatsModuleReferrers
-							moduleStrings={ moduleStrings.referrers }
-							period={ this.props.period }
-							query={ query }
-							summaryUrl={ this.getStatHref( 'referrers', query ) }
-							className={ halfWidthModuleClasses }
-						/>
-
-						<StatsModuleCountries
-							moduleStrings={ moduleStrings.countries }
-							period={ this.props.period }
-							query={ query }
-							summaryUrl={ this.getStatHref( 'countryviews', query ) }
-							className={ clsx( 'stats__flexible-grid-item--full' ) }
-						/>
-
-						{ /* If UTM if supported display the module or update Jetpack plugin card */ }
-						{ supportsUTMStats && ! isOldJetpack && (
-							<StatsModuleUTM
-								siteId={ siteId }
-								period={ this.props.period }
-								query={ query }
-								summaryUrl={ this.getStatHref( 'utm', query ) }
-								summary={ false }
-								className={ halfWidthModuleClasses }
-							/>
-						) }
+					{ ! config.isEnabled( 'stats/paid-wpcom-v3' ) && (
```

Comment:
We probably want to add the feature checks here as otherwise it'll paywall all modules for all sites in staging & development.